src/coreComponents/physicsSolvers/multiphysics/MultiphasePoromechanicsSolver.cpp

```diff
@@ -56,6 +58,11 @@ MultiphasePoromechanicsSolver::MultiphasePoromechanicsSolver( const string & nam
     setInputFlag( InputFlags::OPTIONAL ).
     setDescription( "Constant multiplier of stabilization strength." );
 
+  registerWrapper( viewKeyStruct::isThermalString(), &m_isThermal ).
+    setApplyDefaultValue( 0 ).
+    setInputFlag( InputFlags::OPTIONAL ).
+    setDescription( "Flag indicating whether the problem is thermal or not. Set isThermal=\"1\" to enable the thermal coupling" );
```

Comment:
I do not know the workflow, but why does the end-user have to tell this `MultiphasePoromechanicsSolver` that there will be some thermal simulation. From my review, it looks like it's the `flowSolver`'s responsability?
It looks like there's an overlap?